bitnami/kafka/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kafka
-version: 1.0.2
+version: 1.1.0
```

Comment:
Make sure that the dependencies.yaml is using the proper version of the dependency or it won't get updated.